keel-clouddriver/src/test/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageServiceTests.kt

```diff
@@ -378,48 +348,112 @@ class ImageServiceTests : JUnit5Minutests {
     }
 
     context("given images in varying stages of completeness") {
-      val packageName = "keel"
-      val regions = listOf("us-west-2", "us-east-1")
+      val appVersion = AppVersion.parseName("my-package-0.0.1~rc.97-h98.1c684a9")
+      val regions = setOf("us-west-1", "ap-south-1")
       before {
         every {
-          cloudDriver.namedImages(
-            user = DEFAULT_SERVICE_ACCOUNT,
-            imageName = packageName,
-            account = "test"
+          cloudDriver.namedImages(any(), any(), any())
+        } returns listOf(image1, image6)
```

Comment:
I find these tests pretty hard to reason about with the current naming for the image variables. Consider renaming them to something that helps understand why they're in "varying stages of completeness".

Reply:
Yeah, I thought about completely rewriting this test class since the hardest thing in this entire PR was reasoning about what the fuck it was doing. I felt like the PR was large enough though and that might be best tackled separately.